releasenotes/notes/blackbox-backend-4415919e10a1aa4e.yaml

```diff
@@ -0,0 +1,14 @@
+---
+features:
+  - |
+    Added Blackbox monitoring for backend endpoints by default. Note that this
+    configuration will only work if the Blackbox exporters have access to the
+    backend endpoints.
+issues:
+  - |
+    Backend Blackbox monitoring will not work if the exporter does not have
+    access to the backend OpenStack endpoints. This ususally happens when
```

Comment:
micronit: ususally

Reply:
I can't leave a spelling mistake in good conscience :laughing: